nautobot/docs/user-guides/graphql.md

```diff
@@ -248,6 +248,24 @@ query {
 }
 ```
 
+<div>
+  <details>
+    <summary>View GraphQL Query Results</summary>
+    <img src="../images/graphql/11-graphql-query-06.png">
```

Comment:
Screenshot needs to be updated for the `first`/`limit` change. :-)

Reply:
Oopsies.. I'll get that updated.